package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.9.7",
   "author": "Shopify Inc.",
   "description": "A theme scaffold and command line tool for developing Shopify themes.",
+  "keywords": ["slate", "shopify", "theme", "CLI"],
+  "homepage": "https://github.com/Shopify/slate",
```

Comment:
Because it seems to be the standard for where the code lives, that I've seen on npm. I can change it, if you like?